Components/Alert/alert.html

```diff
@@ -0,0 +1,163 @@
+
+                </div>
+                <a class="side-bar-items" href="../Avatar/avatar.html">Avatar</a>
+                <a class="side-bar-items" href="./alert.html">Alert</a>
+                <a class="side-bar-items" href="../Badge/badge.html">Badge</a>
+                <a class="side-bar-items" href="../Button/button.html">Button</a>
+                <a class="side-bar-items" href="../Cards/card.html">Card</a>
+                <a class="side-bar-items" href="../Responsive-Image/image.html">Image</a>
+                <a class="side-bar-items " href="../Input/input.html">Input</a>
+                <a class="side-bar-items" href="#">List</a>
+                <a class="side-bar-items" href="#">Toast</a>
+            </section>
+        </div>
+        <div class="wrapper">
+
+            <div class="side-bar">
+
+                <span class="side-bar-title">Get Started</span>
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+                <a class="side-bar-items" href="../../Get Started/introduction.html">Introduction</a>
+                <a class="side-bar-items" href="#">Colors</a>
+                <a class="side-bar-items" href="#">Typography</a>
+                <a class="side-bar-items" href="#">Conclusion</a>
+
+                <span class="side-bar-title">Components</span>
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+
+                <a class="side-bar-items" href="../Avatar/avatar.html">Avatar</a>
+                <a class="side-bar-items active" href="./alert.html">Alert</a>
+                <a class="side-bar-items " href="../Badge/badge.html">Badge</a>
+                <a class="side-bar-items " href="../Button/button.html">Button</a>
+                <a class="side-bar-items" href="../Cards/card.html">Card</a>
+                <a class="side-bar-items" href="../Responsive-Image/image.html">Image</a>
+                <a class="side-bar-items " href="../Input/input.html">Input</a>
+                <a class="side-bar-items" href="#">List</a>
+                <a class="side-bar-items" href="#">Toast</a>
+
+            </div>
+
+            <div class="content">
+                <span class="title">Alert</span>
+                <p class="margin-bottom-1 p-content">An alert displays a short, important message in a way that attracts
+                    the user's attention without interrupting the user's task.
+
+                </p>
+                <hr class="margin-bottom-1">
+                <!-- Simple Alert -->
+                <span class="sub-title">Simple Alert</span>
+                <p class="p-content">Simple Alerts can be created using alert class, followed by the variant name.
+                </p>
+
+                <div class="alert-container">
+
+                    <div class="alert primary">
+                        <strong>Primary !</strong>&nbsp This is a Primary Alert - Check it now !
```

Comment:
instead of adding &nbsp you can make one class with padding and margin and font-weight and put primary into it

Reply:
Can't we use &nbsp directly is it the wrong way to follow ?